modules/administration/pages/custom-channels.adoc

```diff
@@ -0,0 +1,78 @@
+. Provide any additional information in the contact details, channel access control, and GPG fields, as required for your environment.
+. Click btn:[Create Channel]
+
+.Procedure: Create a software repository
+
+. In the {productname} {webui}, navigate to menu:Software[Manage > Repositories], and click btn:[Create Repository].
+. On the [guimenu]``Create Repository`` page, give your repository a label (for example, [systemitem]``my-tools-sles15sp1-x86_64-repo``).
+. In the [guimenu]``Repository URL`` field, provide the path to the folder that contains the [path]``repodata`` file (for example, [systemitem]``file:///opt/mytools/``).
+You can use any valid addressing protocol in this field.
+. Uncheck the [guimenu]``Has Signed Metadata?`` check box.
+. Provide any additional information in the SSL fields, as required for your environment.
+. Click btn:[Create Repository]
+. Synchronize your new repository with your custom channel by navigating to menu:Software[Manage > Channels], clicking on the name of your newly created custom channel, and navigating to the [guimenu]``Repositories`` tab.
+. Ensure the repository you want to synchronize is checked, and click btn:[Update Repositories].
+. Navigate to the [guimenu]``Sync`` tab and click btn:[Sync Now] to synchronize immediately.
+You can also set an automated synchronization schedule on this tab.
+
+.Procedure: Add custom channels to an activation key
+
+. In the {productname} {webui}, navigate to menu:Systems[Activation Keys], and select the key you want to add the custom channel to.
+. On the [guiemnu]``Details`` tab, in the [guimenu]``Child Channels`` listing, select the channel to associate.
+You can select multiple channels, if you need to.
+. Click btn:[Update Activation Key].
+
+
+
+== Deleting Custom Channels
+
+You cannot delete {productname} channels.
+Only custom channels can be deleted.
+
+. In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and select the channel you want to delete.
+. Click btn:[Delete software channel].
+. On the [guimenu]``Delete Channel`` page, check the details of the channel you are deleting, and check the [guimenu]``Unsubscribe Systems`` checkbox to remove the custom channel from any systems that might still be subscribed.
+. Click btn:[Delete Channel].
+
+When channels are deleted, the packages that are part of the deleted channel are not automatically removed.
+If you try to update packages that have had their channel deleted, it can cause errors.
+In this case, you will need to manually remove the packages.
+Packages can be manually removed in the {productname} {webui} by going to menu:Systems[Systems > All] and selecting the system, navigating to the menu:States[Packages] tab, and setting the package state to [guimenu]``Removed``.
+Note that removing the package using a command line package manager or the [guimenu]``Software`` menu will not permanently remove the package.
```

Comment:
Not sure what this means. It will remove the package from the system permanently until you install it again or a dependency pulls it in.